client/src/Clusters/TopCharts/TopCharts.js

```diff
@@ -28,7 +28,7 @@ class TopCharts extends Component {
           <h3>{data.subtitle}</h3>
         </div>
         {data.charts && data.charts.map((chart) => {
-            return <div className="col" key={chart.title}>
+            return <div className="col" key={data.id}>
```

Comment:
I just noticed chart doesnt have a unique id either. Can you also add an id in

https://github.com/googleinterns/play-web-step-2020/blob/6f46c567871685309010744ffe898912082f8a5e/src/main/java/com/google/sps/models/Chart.java